aptos-move/framework/aptos-stdlib/sources/math_fixed64.move

```diff
@@ -7,10 +7,20 @@ module aptos_std::math_fixed64 {
 
     /// Abort code on overflow
     const EOVERFLOW_EXP: u64 = 1;
+    /// Abort code on calculation result is negative.
+    const ENEGATIVE_RESULT: u64 = 2;
 
     /// Natural log 2 in 32 bit fixed point
     const LN2: u256 = 12786308645202655660;  // ln(2) in fixed 64 representation
 
+    /// Returns x - y. x must be not less than y.
+    public fun substract(x: FixedPoint64, y: FixedPoint64): FixedPoint64 {
```

Comment:
this should really be in fixed_point64.move and called sub/add

Reply:
SG! Moved to fixed_point64.move.